tensorflow/tools/ci_build/README.md

```diff
@@ -74,3 +74,34 @@ this UI, to see the logs for a failed build:
     2.  In the grid that appears on the right, click on the specific shard,
         run, and attempt to view its log. You can also type the desired shard,
         run, or attempt number in the field above its grid.
+
+### Third party TensorFlow CI
+#### [Mellanox](https://www.mellanox.com/) TensorFlow CI
+
+##### How to start CI
+* Submit special pull request (PR) comment to trigger CI: **bot:mlx:test**
```

Comment:
Would it be better that the tools is triggered ONLY when a bot were AT, say `@bot bot:mlx:test`? In this way, there is no need to keep scanning the PRs.

Reply:
Could you please a bit elaborate on this? Do you mean changing polling logic (track PR creation/changes in the PR comments) into other mechanisms? Webhooks?